src/ui/osx/TogglDesktop/test2/AppDelegate.m

```diff
@@ -1103,8 +1103,6 @@ - (IBAction)onViewChangelogMenuItem:(id)sender
 - (IBAction)onShowMenuItem:(id)sender
 {
 	[self.mainWindowController showWindow:self];
-	[[NSNotificationCenter defaultCenter] postNotificationOnMainThread:kFocusTimer
```

Comment:
I deleted this code since it's **unnecessary** to focus to Timer whenever we open the Time Entry Edit View.

I don't know what is the decision behind the scene, but if this func is executed whenever I click on TimeEntry in TimeEntryList in order to open the Editor.

As a result, sometime, it focus (NSWindow.makeFirstResponder(_)) on Timer Bar, not the Description Text Field in Editor.

Hence, I removed it.

Reply:
The main function of `onShowMenuItem` is to show app window and at that point it has to focus the timer. Maybe it's wrong to call `onShowMenuItem` from the flow of selecting time entry for editing.

Reply:
I revert last commit and do a fix in library at https://github.com/toggl/toggldesktop/pull/2911/commits/ab70ddc51b7c36f86a3d1ef434eb3b6bb062575b

I means that we don't need to call `DisplayApp()` if we just open the Editor. I just tested it. The output is retained. Please let me know your thought @IndrekV 🏄‍♂️